mmocr/datasets/preparers/data_obtainer.py

```diff
@@ -99,10 +99,9 @@ def extract(self,
             delete (bool, optional): Whether to delete the zip file. Defaults
                 to False.
         """
-
-        if not is_archive(src_path):
+        if not is_archive(src_path) and not osp.exists(dst_path):
             # Move the file to the destination folder if it is not a zip
-            shutil.move(src_path, dst_path)
+            shutil.copytree(src_path, dst_path)
```

Comment:
it will raise error when `scr_path` is a file and `dst_path` is a directory

Reply:
Maybe add a check for whether the dst_path exists for it will raise an error if dst_path has existed through `shutil.move`